jest.config.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-console */
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/en/configuration.html
+ */
+import { BeeDebug } from '@ethersphere/bee-js'
+import type { Config } from '@jest/types'
+
+export default async (): Promise<Config.InitialOptions> => {
+  if (!process.env.BEE_POSTAGE) {
```

Comment:
please somewhere notate you can set this environment variable. (either in the doc or at `console.log` or both)